lib/ansible/plugins/action/normal.py

```diff
@@ -33,24 +33,17 @@ def run(self, tmp=None, task_vars=None):
         result = super(ActionModule, self).run(tmp, task_vars)
         del tmp  # tmp no longer has any effect
 
-        if not result.get('skipped'):
+        # FUTURE: better to let _execute_module calculate this internally?
```

Comment:
we can remove this note, since we need this data to remove the tmp dir or not and we don't pass 'task' to execute_module

Reply:
I removed the note in 5d47c40cdff45d75105f478a5ab53233773067b9.